src/main/scala/battleship/WebServer.scala

```diff
@@ -1,21 +1,34 @@
 package battleship
 
-import akka.actor.ActorSystem
+import akka.actor.{ActorRef, ActorSystem}
 import akka.http.scaladsl.Http
 import akka.http.scaladsl.server.Directives
 import akka.stream.ActorMaterializer
-import battleship.routes.{BaseRoutes, SimpleRoutes}
+import battleship.GameActor.props
+import battleship.routes._
 
 import scala.io.StdIn
 
-object WebServer extends Directives with SimpleRoutes {
+class BaseRoute (inputGameActor: ActorRef) extends BaseRoutes with ActorRoutes with SimpleRoutes {
+  var gameActor = inputGameActor
```

Comment:
There was at some point in it's short lived history, but not a good one, I changed it.